docs/xknx.md

```diff
@@ -44,6 +45,7 @@ The constructor of the XKNX object takes several parameters:
 * `rate_limit` in telegrams per second - can be used to limit the outgoing traffic to the KNX/IP interface. The default value is 20 packets per second.
 * `multicast_group` is the multicast IP address - can be used to override the default multicast address (`224.0.23.12`)
 * `multicast_port` is the multicast port - can be used to override the default multicast port (`3671`)
+* `log_to_file` is a boolean - when set to True we log to a dedicated file `xknx.log`
```

Comment:
We should enable users to pass a directory where to write the loggiles at. 
Especially in Docker environments the Loggien could go lost.

Reply:
Yea good point. Should we have multiple logfiles or only one? It's only one right now with each type having there own column (%(name) is the logger name in the formatter)

Reply:
I'd prefer one file so the rows are in timed order.